makefiles/Makefile.sim

```diff
@@ -27,13 +27,14 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ##############################################################################
 
-# Backwards-compatibility wrapper for people using
-# "include $COCOTB/makefiles/Makefile.sim" in their Makefile.
+COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
+ifeq ($(COCOTB_CONFIG_CMD),)
+    $(error 'Cocotb is not installed (cocotb-config not found). \
+             Please refer to https://cocotb.readthedocs.io/en/latest/quickstart.html#installing-cocotb \
```

Comment:
Does this link mention adding it to the path?

Reply:
No. I originally had this error in  `cocotb/share/makefiles/Makefile.inc`. Maybe it should go there also? 

The problem may be for people who use '--user' (that is not in our documentation).

Reply:
It would be ok to make an issue for me and add NOTE in the documentation?

Reply:
I've also had this problem on systems where python itself is not on the path, typically Windows if the "add to path" box in the installer is not checked.

My own system is set up this way. Python 3.5 is on the path, but cocotb is installed into 3.8 which is not.

All I'd suggest is replacing "not installed" with "not installed (or installed but not on the PATH)

Reply:
You are right. I will do a patch immediately. Sorry.